spec/src/main/asciidoc/app-programming-model.adoc

```diff
@@ -40,12 +44,6 @@ public int getQueueSize() {
 }
 ----
 
-NOTE: The programming API follows Dropwizard Metrics 3.2.3 API, but with local changes.
```

Comment:
while MP Metrics has diverged somewhat from Dropwizard Metrics, this seemed like a useful comment to 1) give credit to Dropwizard Metrics, and 2) recognize that the similarities between MP metrics and DW Metrics were intentional.

Reply:
Maybe add back but say it was "inspired by" instead of "follows"?

Reply:
Re-added.